pkg/labels/selector.go

```diff
@@ -72,10 +72,13 @@ const (
 	ExistsOperator       Operator = "exists"
 )
 
-//LabelSelector is a list of Requirements.
-type LabelSelector []Requirement
+func NewSelector() Selector {
```

Comment:
+1, you can always call the variadic form without args if you want to use it that way.